src/timeoutLink.ts

```diff
@@ -72,6 +72,17 @@ export default class TimeoutLink extends ApolloLink {
         observer.error(new Error('Timeout exceeded'));
         subscription.unsubscribe();
       }, ctxTimeout || this.timeout);
+      
+      let ctxRef = operation.getContext().timeoutRef
+
+      if (ctxRef) {
+        ctxRef({
+          unsubscribe: () => {
+            clearTimeout(timer)
+            subscription.unsubscribe()
+          }
+        })
+      }
```

Comment:
Hi Anifowoshe. Thanks for the contribution! This change looks fine to me but would you mind adding semi-colons to the end of the lines in order to be consistent with the rest of the file?

Reply:
Okay thanks .... Would add that right away